WebfrontCore/Views/Client/Privileged/Index.cshtml

```diff
@@ -2,7 +2,9 @@
 
 <h4 class="pb-3 text-center ">@ViewBag.Title</h4>
 
-<div class="row border-bottom">
+@if (ViewBag.Authorized)
```

Comment:
Is probably more effective to just conditionally add page in `_Layout.cshtml`
![image](https://user-images.githubusercontent.com/6732771/103464827-59faa480-4cfc-11eb-83bf-67f2e4c0ac20.png)
However incase someone tried to access the URL anyway, in `ClientController.cs->PrivilegedAsync`, you can return a 401 if they're not authorized